### DISCUSSION TITLE
Simplified Project Contribution Information OUTREACHY 2024!

### DISCUSSION THREAD

**Discussion by Kingjhenny** [camicroscope/caMicroscope#686] [General]
**Here's a simpler summary of the project contribution information:**

**1. Getting Started:**

-  Start by forking the code repository you want to contribute to.
- Create a new branch for each contribution and make your changes there.
- Follow coding standards and guidelines while making changes.
- Test your changes thoroughly before submitting them.

**2.  Development Workflow:**

- Changes are made through pull requests (PRs).
- Open a PR against the original repository's main branch once your changes are ready.
- Be prepared to address feedback from the team during code review.
- Once approved, your changes will be merged into the main repository.

**3. Best Practices:**

- Work on topic branches from your fork for smoother contribution.
- Keep your fork's master or develop branch up-to-date with the original repository.
- Refer to the Triangle Workflows Guide for Git usage guidance.

**4. Reporting Issues:**

- If you encounter issues, create issues and PRs against the relevant repository.
- Provide clear steps to reproduce the issue and include screenshots if possible.
- Share device, platform, and version information along with any relevant notes.

**5. Requesting Features:**

- Propose new features or improvements with clear summaries and background information.
- Describe use cases and expected impacts of the proposed feature.
- Provide any implementation ideas or technical details to support your proposal.

**6. Contribution Guidelines:**

- Review the community guide to ensure your contributions align with project goals and standards.
- Guidelines cover topics like coding standards, testing, and the review process.

Remember to follow these guidelines to contribute effectively to the caMicroscope project.

**Comment by birm** [comment]
Good summary!!

**Reply by Kingjhenny** [reply]
Thank you!! @birm

**Reply by blessdamy** [reply]
Thank you @Kingjhenny and @birm
